src/azure-cli/azure/cli/command_modules/synapse/operations/workspace.py

```diff
@@ -15,12 +16,19 @@ def list_workspaces(cmd, client, resource_group_name=None):
 
 
 def create_workspace(cmd, client, resource_group_name, workspace_name, storage_account, file_system,
-                     sql_admin_login_user, sql_admin_login_password, location=None, enable_managed_virtual_network=None,
-                     tags=None, no_wait=False):
+                     sql_admin_login_user, sql_admin_login_password, location=None, key_name="default", key_identifier=None, enable_managed_virtual_network=None,
+                     allowed_aad_tenant_ids=None, tags=None, no_wait=False):
     identity_type = "SystemAssigned"
     identity = ManagedIdentity(type=identity_type)
     account_url = "https://{}.dfs.{}".format(storage_account, cmd.cli_ctx.cloud.suffixes.storage_endpoint)
     default_data_lake_storage = DataLakeStorageAccountDetails(account_url=account_url, filesystem=file_system)
+    if str(key_identifier).endswith('/'):
+        key_identifier = key_identifier[:-1]
+    workspace_key_detail = WorkspaceKeyDetails(name=key_name, key_vault_url=key_identifier)
+    encryption = EncryptionDetails(cmk=CustomerManagedKeyDetails(key=workspace_key_detail))
+    managed_virtual_network_settings = None
+    if enable_managed_virtual_network:
+        managed_virtual_network_settings = ManagedVirtualNetworkSettings(preventDataExfiltration=True, allowed_aad_tenant_ids_for_linking=allowed_aad_tenant_ids)
```

Comment:
>preventDataExfiltration=True [](start = 73, length = 28)

I remember from portal, this property can be `False`

Reply:
Will add a parameter to indicate whether enable data exfiltration.

`--enable_data_exfiltration` is fine?

Reply:
Yes, sounds good.